src/graphql/data/nft/NftActivity.ts

```diff
@@ -121,7 +121,7 @@ export function useNftActivity(filter: NftActivityFilterInput, first?: number, f
           toAddress: activity.toAddress,
           transactionHash: activity.transactionHash,
           orderStatus: activity.orderStatus,
-          price: wrapScientificNotation(activity.price?.value),
+          price: wrapScientificNotation(activity.price?.value ?? 0),
```

Comment:
Does `wrapScientificNotation` allow `undefined` to be passed in? The typing should also be updated.